hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/multitenant/AccessPolicy.java

```diff
@@ -18,6 +18,7 @@
 
```

Comment:
I attempted and figured it would be another >500 lines of +/- if we move the whole logic from AccessPolicy to MultiTenantAccessController in this PR.

I suggest we do the refactoring in another separate PR for easier review.